src/domain/core/src/entities/dataset.rs

```diff
@@ -61,6 +61,25 @@ pub trait Dataset: Send + Sync {
         checkpoint: Option<&OwnedFile>,
     ) -> Result<ExecuteTransform, InternalError>;
 
+    /// Helper function to prepare [AddDaat] event parameters
```

Comment:
Typo: `AddDaat` => `AddData`.
`AddDaat` sounds very much like Dutch language :-)

Reply:
Just wanted bring some Dutch in project.
Fixed)